service/http.directory/src/main/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceHandler.java

```diff
@@ -148,9 +151,15 @@ protected void doSessionOpened(HttpAcceptSession session) throws Exception {
         }
 
         if (!underBaseDir) {
-            reportError(session, HttpStatus.CLIENT_BAD_REQUEST);
-            session.close(false);
-            return;
+            boolean isSymLink = Files.isSymbolicLink(requestFile.toPath());
```

Comment:
Can we add a test where the path contains a symlink that is not in the sandboxed area?